apps/backend/src/database/datasource.ts

```diff
@@ -24,5 +27,7 @@ export default new DataSource({
     LocationEntity
   ],
   migrations: [__dirname + '/migrations/**/*{.ts,.js}'],
-  synchronize: false
+  synchronize: false,
+  logging: logLevels,
+  logger: dbLogger
```

Comment:
could be conditional insert using spread operator

Reply:
I'm not sure what you mean here

Reply:
(I know what a spread operator and a conditional is, I'm just unclear on how you want them used here)

Reply:
@vyasworks : Are you referring to another place in the code? I don't see where the spread operator could be used in this definition, so I'm going to approve this PR and we can fix it later if there's a better way to do it.

Reply:
yes, I think my PR on running migrations will include it. so good for now.